README.md

```diff
@@ -16,7 +16,7 @@
 <meta name="msapplication-TileImage" content="/images/favicon/ms-icon-144x144.png">
 <meta name="theme-color" content="#ffffff">
 
-<img src="https://github.com/vorburger/minecraft-storeys-maker/raw/master/logo/oasis.learn.study-Minecraft-Scratch-HighRes.png" width="230" align="right"/> 
+<img src="https://github.com/vorburger/minecraft-storeys-maker/raw/master/logo/oasis.learn.study-Minecraft-Scratch-HighRes.png" width="230" align="right"/>
```

Comment:
why are you changing the line ending here?

Reply:
the line ending was unneccesary

Reply:
oh, sorry, you are right of course - will merge